cylc/flow/tests/host_select.py

```diff
@@ -0,0 +1,186 @@
+"""Test the cylc.flow.host_select module.
+
+NOTE: these are functional tests, for unit tests see the docstrings in
```

Comment:
I'm not sure that the doctests are a full set of unit tests.

Reply:
Together they provide 100% coverage.